code/BatchActions/CMSBatchAction_Archive.php

```diff
@@ -27,6 +28,6 @@ public function run(SS_List $pages): HTTPResponse
 
     public function applicablePages($ids)
     {
-        return $this->applicablePagesHelper($ids, 'canArchive');
+        return Deprecation::withNoReplacement(fn() => $this->applicablePagesHelper($ids, 'canArchive'));
```

Comment:
```suggestion
        // canArchive() is deprecated, not $this->applicablePagesHelper()
        return Deprecation::withNoReplacement(fn() => $this->applicablePagesHelper($ids, 'canArchive'));
```

Reply:
I don't see the point of the comment but I added it to avoid ping pong.